go.sum

```diff
@@ -0,0 +1,53 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
too many libraries here ;) just use go mod tiny to remove redundant dependencies.